pkg/metrics/metrics.go

```diff
@@ -26,13 +30,24 @@ const (
 )
 
 var (
-	metricPlaceholder = prometheus.NewGaugeVec(prometheus.GaugeOpts{
-		Name: "placeholder",
-		Help: "Placeholder",
+	metricPagerDutyCreateFailure = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "metricPagerDutyCreateFailure",
+		Help: "metricPagerDutyCreateFailure",
+	}, []string{"name"})
+
+	metricPagerDutyDeleteFailure = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "metricPagerDutyDeleteFailure",
+		Help: "metricPagerDutyDeleteFailure",
+	}, []string{"name"})
+
+	metricPagerDutyHeartbeat = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "metricPagerDutyHeartbeat",
+		Help: "Metric for heartbeating the Dead Man's Snitch api",
 	}, []string{"name"})
 
 	metricsList = []prometheus.Collector{
-		metricPlaceholder,
+		metricPagerDutyCreateFailure,
+		metricPagerDutyDeleteFailure,
```

Comment:
You will also need to put the heartbeat metric in here

Reply:
Updated.